### DISCUSSION TITLE
Add mail_class and tracking_url to receipt endpoints

### DISCUSSION THREAD

**Discussion by emergingdzns** [etsy/open-api#821] [Ideas]
One of the differences between V2 and V3 is the data returned for the receipt. In V2 the api returned "mail_class" which would provide the class chosen for the carrier (USPS - first class, media, 3-day etc, UPS - ground, next-day etc). At the moment, it doesn't seem like we get this kind of info. If it is provided, can you clarify how we would derive this from a receipt?

Also, if at all possible, and whenever possible, can you also provide the tracking URL for the shipment?

**Comment by Gareth064** [comment]
I believe this info is found inside the nested shipping object of the receipt. If you don't see it there then see if it's at the transaction level
